README.md

```diff
@@ -25,33 +26,35 @@ or
 ## Usage
 
 ```javascript
-const schema = FluentSchema()
+const userSchema = FluentSchema()
```

Comment:
Can you include a require statement here?

Reply:
good point 👍